src/fsharp/FSharp.Core/FSharp.Core.fsproj

```
@@ -17,7 +17,8 @@
     <PackageId>FSharp.Core</PackageId>
     <NuspecFile>FSharp.Core.nuspec</NuspecFile>
     <IsPackable>true</IsPackable>
-    <PackageDescription>FSharp.Core redistributables from Visual F# Tools version $(FSPackageMajorVersion) For F# $(FSCoreMajorVersion).  Contains code from the F# Software Foundation.</PackageDescription>
+    <PackageDescription>FSharp.Core redistributables from Visual F# Tools version $(FSCorePackageVersion) For F# $(FSLanguageVersion).  Contains code from the F# Software Foundation.</PackageDescription>
```

```suggestion
    <PackageDescription>FSharp.Core redistributables from F# Tools version $(FSCorePackageVersion) For F# $(FSLanguageVersion).  Contains code from the F# Software Foundation.</PackageDescription>
```

Let's finally drop the "Visual" from here :)

---

The fcs release notes are already in, they are at the bottom of the release-notes file.
Yes I can yeet the Visual from Visual FSharp.

---

Yee